src/ripple/server/tests/Server.test.cpp

```diff
@@ -48,9 +48,9 @@ class Server_test : public beast::unit_test::suite
 
     public:
         TestThread()
-            : work_(boost::in_place(std::ref(io_service_)))
-            , thread_([&]() { this->io_service_.run(); })
+            : thread_([&]() { this->io_service_.run(); })
         {
+            work_.emplace (io_service_);
```

Comment:
This ain't gonna work, the call to `work_.emplace` must happen before the call to `io_service_.run`.